src/components/SatelliteManifestPanel/__tests__/SatelliteManifestPanel.test.tsx

```diff
@@ -186,48 +188,51 @@ describe('Satellite Manifest Panel', () => {
     expect(container).toMatchSnapshot();
   });
 
-  it('opens the delete popup from clicking the kebab menu', () => {
-    (useSatelliteVersions as jest.Mock).mockReturnValue({
-      body: [] as SatelliteVersion[]
-    });
+  describe('when the user does not have write permissions', () => {
+    def('canWriteManifests', () => false);
 
-    const { getByLabelText, getByText } = render(
-      <QueryClientProvider client={queryClient}>
-        <SatelliteManifestPanel {...get('props')} />
-      </QueryClientProvider>
-    );
+    it(' it does not open the delete popup from clicking the kebab menu', () => {
+      (useSatelliteVersions as jest.Mock).mockReturnValue({
+        body: [] as SatelliteVersion[]
+      });
 
-    fireEvent.click(getByLabelText('Actions'));
-    fireEvent.click(getByText('Delete'));
+      const { getByLabelText, getByText } = render(
+        <QueryClientProvider client={queryClient}>
+          <SatelliteManifestPanel {...get('props')} />
+        </QueryClientProvider>
+      );
+      fireEvent.click(getByLabelText('Actions'));
+      fireEvent.click(getByText('Delete'));
 
-    expect(
-      screen.queryByText('Deleting a manifest is STRONGLY discouraged. Deleting a manifest will:')
-    ).toBeInTheDocument();
-  });
+      expect(
+        screen.queryByText('Deleting a manifest is STRONGLY discouraged. Deleting a manifest will:')
+      ).toMatchSnapshot();
```

Comment:
Why did this change to `toMatchSnapshot`? That kind of changes what this is actually testing. Also if this test says the popup should not be present, why are we still querying by a message on it?

Reply:
If we want to keep that query in we should ensure that it is _not_ in the document